src/tokens/basecamp/metadata.cairo

```diff
@@ -0,0 +1,105 @@
+# SPDX-License-Identifier: MIT
+
+%lang starknet
+
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin
+from starkware.cairo.common.uint256 import Uint256
+
+from openzeppelin.token.erc721.library import _exists
+
+from openzeppelin.introspection.ERC165 import ERC165
+
+from src.utils.ShortString import uint256_to_ss
+from src.utils.Array import concat_arr
```

Comment:
These are helper functions that I borrowed and modified from https://github.com/l-henri/starknet-erc721